manticore/native/cpu/x86.py

```diff
@@ -5701,7 +5701,8 @@ def MOVHPD(cpu, dest, src):
         :param src: source operand.
         """
         if src.size == 128:
-            assert dest.size == 64
+            print(dest.size)
```

Comment:
Why is this `print` here? Just something that snuck through while debugging? :)

Reply:
Ah yeah, a capstone bug lead me down that route. That function normally never gets called the opcode is manually changed to movs due to a mnemonic conflict it seems. Where one movhpd says it can't write to memory and the other can. If I remember correctly. I'll fix that after this meeting.